Source/WebInspectorUI/UserInterface/Views/ComputedStyleDetailsPanel.js

```diff
@@ -225,25 +230,24 @@ WI.ComputedStyleDetailsPanel = class ComputedStyleDetailsPanel extends WI.StyleD
             if (styleSection === this._computedStyleSection)
                 continue;
 
-            // WI.ComputedStyleSection is a view but its element is attached to a WI.DetailsSection which isn't. Reparent it before removing.
-            // FIXME: <https://webkit.org/b/152269> - Web Inspector: Convert DetailsSection classes to use View
-            this.element.appendChild(styleSection.element);
-            this.removeSubview(styleSection);
-            styleSection.element.remove();
-            this._detailsSectionByStyleSectionMap.delete(styleSection);
-            detailsSection.element.remove();
+            this._removeVariablesStyleSection(styleSection);
         }
 
-        this._variablesStyleSectionForGroupTypeMap.clear();
+        if (!this.nodeStyles.computedStyle)
```

Comment:
Is this actually possible?  Or is this a proxy for checking `this.nodeStyles.needsRefresh`?

Reply:
`WI.DOMNodeStyles.computedStyle` starts off as `null`. It gets populated in a promise:
[Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js?L320-323](https://sourcegraph.com/github.com/WebKit/WebKit@29b13de20953a2bdfdb1f5dfa7bf3c856d120009/-/blob/Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js?L320-323)

I guess checking `this.nodeStyles.needsRefresh` would also be fine.